tuist/tuist
# Strict and xcode warnings in inspect command

### Short description 📝

Added strict mode for inspect command which is classic for lint utils. [Here](https://github.com/realm/SwiftLint/blob/dfd19bddc20538e4b151927556ea542c8fb021db/Source/SwiftLintCore/Models/Configuration.swift#L39) and [here](https://github.com/peripheryapp/periphery/blob/41e5034d81472e911983dc9191391f120192bbc0/Sources/Shared/Configuration.swift#L99). We need this mode, because when user try to launch tuist inspect from xcode, the build will stop after executing our script, but in some cases user wants to execute few scripts in a row or just use our script as a helper, but not as stopper for build

Also added xcode flag which is adding new format of output, that xcode can read. 

### How to test the changes locally 🧐

To test strict just pass --strict as a parametr

To test xcode launch Xcode Project -> Add run script Phase ->swift run --package-path <path-to-your-tuist> tuist lint implicit-imports --path <path-to-test-project> --xcode

> Include a set of steps for the reviewer to test the changes locally (see [the documentation](https://docs.tuist.io/contributors/get-started) for reference).

### Contributor checklist ✅

- [✅ ] The code has been linted using run `mise run lint-fix`
- [✅ ] The change is tested via unit testing or acceptance testing, or both
- [ ✅] The title of the PR is formulated in a way that is usable as a changelog entry

### Reviewer checklist ✅
